src/index.scss

```diff
@@ -22,7 +22,7 @@ $mobile-width: 720px;
   @media(max-width: #{$mobile-width}) { @content; }
 }
 
-//override sass min() & max(), use css
+//override sass min() & max() fn's, use css's
 @function min($numbers...) {
```

Comment:
Just to clarify: why use the CSS ones over SASS?

Reply:
IIRC, the SASS ones don't work with multiple units which is why I initially had the weird unquote() thing